scanapi/settings.py

```diff
@@ -23,6 +23,7 @@ def __init__(self):
         self["output_path"] = None
         self["template"] = None
         self["no_report"] = False
+        self["open_browser"] = False
 
         super(dict, self).__init__()
```

Comment:
Can you update the super call? From `super(dict, self).__init__()`  -> `super().__init__()` that way I guess static analysis CI would pass.